lib/core/src/model.rs

```diff
@@ -787,6 +787,9 @@ pub enum PayAmount {
         asset_id: String,
         receiver_amount: f64,
         estimate_asset_fees: Option<bool>,
+        /// Specifies whether or not to use the wallet's funds to execute the payment. Defaults to true.
+        /// If false, it will try swapping the asset via the [Side Swap Service](crate::side_swap::api::SideSwapService)
+        use_wallet_funds: Option<bool>,
```

Comment:
Could we change this to `use_assets` or `use_wallet_assets`? To me `use_wallet_funds` doesn't make 100% sense as L-BTC is also wallet funds